### DISCUSSION TITLE
Weird Mode container

### DISCUSSION THREAD

**Discussion by VSevagen** [pradyunsg/furo#210] [Potential Issue] [answered]
Hi, I'm not sure this is really an issue but I couldn't find a solution for it. Moroever this happens only in chrome. I opened the link in firefox and everything looks fine.

![Screenshot_2021-07-30_11-12-09](https://user-images.githubusercontent.com/43094266/127617593-133b68d8-355c-479a-9b8d-16e67a5af8be.png)

**Comment by pradyunsg** [comment, marked as answer]
Hah. Could it be a stale cached CSS file? Does ctrl+shift+R solve it?

**Reply by VSevagen** [reply]
Could be. It got resolve with normal reload after a while so I couldn't test ctrl+shift+ R.

**Reply by pradyunsg** [reply]
Yea, that looks like weird caching.

11dc2eebe65a64cfdecbabbfaa7f75c66fce5a21 should prevent this from happening again.

**Reply by cathaysia** [reply]
I had this problem also: https://github.com/pradyunsg/furo/discussions/211
This can be solved by refresh after a little times.

**Reply by pradyunsg** [reply]
Released beta41, which contains 11dc2ee.
